v1/bindings/dotnetcore/dotnet-core-binding/Microsoft.Azure.Devices.Gateway.Tests/Microsoft.Azure.Devices.Gateway.Tests.csproj

```diff
@@ -7,7 +7,7 @@
     <PackageId>Microsoft.Azure.Devices.Gateway.Tests</PackageId>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <PackageTargetFallback>$(PackageTargetFallback);dotnet5.4;portable-net451+win8</PackageTargetFallback>
-    <RuntimeFrameworkVersion>1.1.1</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion>1.1.11</RuntimeFrameworkVersion>
```

Comment:
Any reason you picked 1.1.11? They released one more point on the 1.1.x line -> 1.1.12. There's also Net core 2.x out now since Net Core 3.x is coming next month.

Reply:
Looks like there is a NuGet tampering fix that affects linux that is fixed in 1.1.12 https://github.com/dotnet/core/blob/master/release-notes/1.1/1.1.12/1.1.12.md

Reply:
I only upgraded to get onto the current build servers. 1.1.12 is the natural choice, but IIRC it wasn't available in the default location on the agent--1.1.11 was. I could easily get 1.1.12 by using the installer task, but it gets installed at a different location, and our code/build isn't set up to find it there.